From a19b19b90e3aa95731c4f9b590363ab70a0da5f2 Mon Sep 17 00:00:00 2001
From: EricYang-wiwynn <Eric_Yang@wiwynn.com>
Date: Mon, 1 Apr 2024 15:05:58 +0800
Subject: [PATCH 1010/1022] Revise duty cycle for SMB9 and SMB10

ARM: dts: aspeed: yosemite4:
Revise duty cycle for SMB9 and SMB10 to 40:60
To meet 400kHz-i2c clock low time spec (> 1.3 us).

https://lore.kernel.org/all/20240401090509.2338027-1-Delphine_CC_Chiu@Wiwynn.com/
---
 arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts
index b9c10526c2a2..b6966314f1e6 100644
--- a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts
@@ -545,6 +545,7 @@ &i2c8 {
 	#size-cells = <0>;
 	status = "okay";
 	bus-frequency = <400000>;
+        i2c-clk-high-min-percent = <40>;
 	i2c-mux@70 {
 		compatible = "nxp,pca9544";
 		reg = <0x70>;
@@ -667,6 +668,7 @@ &i2c9 {
 	#size-cells = <0>;
 	status = "okay";
 	bus-frequency = <400000>;
+        i2c-clk-high-min-percent = <40>;
 	i2c-mux@71 {
 		compatible = "nxp,pca9544";
 		reg = <0x71>;
-- 
2.44.2

